### PR TITLE
Resume: Remember Playlist Position

### PR DESCRIPTION
**Pull request by criztovyl**
This is a WIP PR for implementing resuming playlists: #6 

So far I was able to save the current playlist status to the resume file and load it into the `resume` struct again, but I am struggling to set the current playlist and it's playing track.
Maybe I will find out on my own what i am missing at the moment, but any help is highly appreciated. ^^

Opening the PR early for better feedback. :)

### CODE REVIEW

File: pl.c [hunk 1]
```diff
@@ -921,13 +926,29 @@ const char *pl_marked_pl_name(void)
...
+const char *pl_playing_pl_name(void)
+{
+	return pl_playing->name;
+}
+
+void pl_resume(const char *name, struct track_info *ti)
+{
+
+	struct playlist *pl = pl_get_pl_by_name(name);
+	// ...
```
Comment (criztovyl) [inline pl.c]: This is where I am struggeling. ATM I am trying `pl_play_track()`, but I get `window_set_sel: BUG: iter_is_empty(&win->top)`, so I guess I am missing some initializations.

**Comment by pgaskin** [comment]
@criztovyl, are you still working on this?

**Comment by criztovyl** [comment]
Unfortunatey not, no.

**Comment by nefthy** [comment]
Closing due to inactivity. Please feel free to reopen or create a new one.

**Comment by pgaskin** [comment]
Fixed in #1325.

**Pull request closed without merging**